diff --git a/libreoffice-core/desktop/source/lib/init.cxx b/libreoffice-core/desktop/source/lib/init.cxx
index eb1df9ce5e..26f16ece4f 100644
--- a/libreoffice-core/desktop/source/lib/init.cxx
+++ b/libreoffice-core/desktop/source/lib/init.cxx
@@ -3458,6 +3458,10 @@ static int doc_saveAs(LibreOfficeKitDocument* pThis, const char* sUrl, const cha
         if (pdfVer)
             aFilterDataMap["SelectPdfVersion"] <<= pdfVer;
 
+
+        aFilterDataMap["ExportNotes"] <<= false;
+        aFilterDataMap["ExportNotesInMargin"] <<= true;
+
         if (!aFilterDataMap.empty())
         {
             aSaveMediaDescriptor["FilterData"] <<= aFilterDataMap.getAsConstPropertyValueList();
